web_monitoring/differs.py

```diff
@@ -1,13 +1,14 @@
-from bs4 import BeautifulSoup
-import diff_match_patch
+from bs4 import BeautifulSoup   
```

Comment:
Also really minor, but we should clean up the spaces at the end of the line here. Do you use `editorconfig` at all? It can really help if your editor supports it.